src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -275,31 +272,41 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
             emit ProjectCollateralFeeAdded(projectFeeAmount);
         }
 
-        // Calculate mint amount based on upstream formula
-        uint issuanceMintAmount = _issueTokensFormulaWrapper(netDeposit);
-        totalIssuanceTokenMinted = issuanceMintAmount;
+        // Calculate token amount based on upstream formula
+        uint issuanceTokenAmount = _issueTokensFormulaWrapper(netDeposit);
+        totalIssuanceTokenMinted = issuanceTokenAmount;
 
         // Get net amount, protocol and project fee amounts. Currently there is no issuance project
         // fee enabled
-        (issuanceMintAmount, protocolFeeAmount, /* projectFeeAmount */ ) =
+        (issuanceTokenAmount, protocolFeeAmount, /* projectFeeAmount */ ) =
         _calculateNetAndSplitFees(
-            issuanceMintAmount, issuanceBuyFeePercentage, 0
+            issuanceTokenAmount, issuanceBuyFeePercentage, 0
         );
         // collect protocol fee on outgoing issuance token
         _processProtocolFeeViaMinting(issuanceTreasury, protocolFeeAmount);
 
-        // Revert when the mint amount is lower than minimum amount the user expects
-        if (issuanceMintAmount < _minAmountOut) {
+        // Revert when the token amount is lower than minimum amount the user expects
+        if (issuanceTokenAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Mint tokens to address
-        _mint(_receiver, issuanceMintAmount);
+
+        // Use virtual function to distribute issuance tokens
+        _handleIssuanceTokenAfterBuy(_receiver, issuanceTokenAmount);
+
         // Emit event
```

Comment:
Was here before already, but this comment also doesn't win a prize of being too helpful, let's just remove it then :D

Reply:
Done